src/test/java/taskFinal/test/TestBase.java

```diff
@@ -0,0 +1,23 @@
+package taskFinal.test;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+import taskFinal.util.SingleWebDriver;
+
+import java.net.MalformedURLException;
+
+public abstract class TestBase {
+    protected WebDriver driver;
+    private String url;
+
+    public TestBase(String url) {
+        this.url = url;
+    }
+
+    @BeforeEach
+    protected void setUp() throws MalformedURLException {
+        driver = SingleWebDriver.getWebDriverInstance().getTargetWebDriver();
+        driver.get(this.url);
+        driver.manage().window().fullscreen();
```

Comment:
It is better to have configuration in SingleWebDriver class

Reply:
Done